src/components/PokemonDetails.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+export default function PokemonDetails({ pokemonDetails, pokemonName }) {
+  return (
+    <div className="card card-body" >
+      <h5 className="card-title">Pokemon details</h5>
+      <h6 className="card-title">{pokemonName}</h6>
+      {pokemonDetails.map(detail => (
```

Comment:
Here you can also destructure the props that you need from detail, not the same case than above because here you're not sending the whole object to a function, something like:

```JSX
 pokemonDetails.map(({ stat, base_stat })) => ( ...
```

This way you can just get just what's needed and avoid break the demeter law:
https://devexperto.com/ley-de-demeter/

Reply:
Thanks, that is a really good point